libs/gi/formula/src/data/common/conds.ts

```diff
@@ -0,0 +1,8 @@
+import { allBoolConditionals } from '../util'
+
+export const { enemyFrozen } = allBoolConditionals('static', 'both')
+export const {
+  hasShield,
+  nearbyDendro1, // Burning, Quicken, Bloom
```

Comment:
nearbyDendro1 and nearbyDendro2 should be moved to resonance. we can leave the other 2

Reply:
The names still can't collide with other `static` conditionals (the first argument in `allBoolConds`), even if they're declared on different sheets. Wouldn't it be confusing to put them on different files?

Reply:
prepend name with `reso_` or something then. these are still resonance specific so i wouldnt want to export them from common

Reply:
then we can add to `reso` sheet instead